designer/server/src/main/scala/pl/touk/nussknacker/ui/api/description/NodesApiEndpoints.scala

```diff
@@ -1472,6 +1542,7 @@ object NodesApiEndpoints {
     sealed trait NodesError
 
     object NodesError {
+      final case class FetchLatestRecordsError(msg: String)             extends NodesError
```

Comment:
:/ It would be nice to have all business cases such as node compilation error, source doesn't support data preview ... etc. here. Can you provide it? At least let's add a TODO for that